HISTORY.rst

```diff
@@ -2,6 +2,21 @@
 History
 =======
 
+2.0.13 (2018-10-02)
+-------------------
+* Happy Spooky October! It's unlucky release 2.0.13, with some scary-cool improvements. Just to show you how ramped up our RelEng team is now, this release had TWENTY THREE pull requests in 12 days! From all four of your friendly SFDO Release Engineering committers. Thanks so much for continuing to use CCI for all your Salesforce automation needs.
+* NEW FLOW: ci_beta_dependencies installs the latest beta of project dependencies and run tests. Includes task error when running against non-scratch orgs.
+* NEW TASK: ReportPushFailures pulls a list of Package Push Upgrade Request failures after a push attempt, including grouping by important factors. Since this task is for packaging orgs only, you'll need to wire it up in your local ~.cumulusci/cumulusci.yml.
```

Comment:
Nah, we did wire it up. https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml#L144